### ISSUE TITLE
esupy 'dependency conflict'

### ISSUE THREAD

**Issue by dpgraham4401** [USEPA/flowsa#103]
Howdy,
I'm receiving the following error when following the installation instructions. 
 
```
Collecting esupy@ git+git://github.com/USEPA/esupy@v0.1.7#egg=esupy
  Cloning git://github.com/USEPA/esupy (to revision v0.1.7) to /tmp/pip-install-v0p7lwqy/esupy_0e824dc63ad04367b1ae3e24b3bdde5d
  Running command git clone -q git://github.com/USEPA/esupy /tmp/pip-install-v0p7lwqy/esupy_0e824dc63ad04367b1ae3e24b3bdde5d
  Running command git checkout -q c04efa5aefc82a317776a2b2b1b20fae01b5fce7
INFO: pip is looking at multiple versions of fedelemflowlist to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of esupy to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of flowsa to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install flowsa and flowsa==0.2.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    flowsa 0.2.1 depends on esupy 0.1.7 (from git+https://github.com/USEPA/esupy@v0.1.7#egg=esupy)
    stewi 0.9.9 depends on esupy 0.1.7 (from git+git://github.com/USEPA/esupy@v0.1.7#egg=esupy)

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

```
steps to recreate 
```
$ mkdir flowsa-test
$ python3 -m venv flowsa-venv
$ source flowsa-venv/bin/activate
$ pip install git+https://github.com/USEPA/flowsa
```

pip version 21.0.1
python version 3.9.6
Following the installation instructions outside of a virtual environment was also unsuccessful. 

I was able to successfully install a fork by changing the 'git' to 'https' in setup.py but then I ran into more import problems and wasn't sure if i was causing more problems than I was fixing.

**Comment by catherinebirney** [comment]
Thanks @dpgraham4401. Will fix this error today with a new release of flowsa.

**Comment by bl-young** [comment]
@catherinebirney I didn't realize that this would cause an issue:
    flowsa 0.2.1 depends on esupy 0.1.7 (from git+**_https:_**//github.com/USEPA/esupy@v0.1.7#egg=esupy)
    stewi 0.9.9 depends on esupy 0.1.7 (from git+_**git:**_//github.com/USEPA/esupy@v0.1.7#egg=esupy)

I believe git is correct but i'm not positive

**Comment by bl-young** [comment]
@catherinebirney ok i've released 0.10.0 of stewi, lets try pointing to that to see if this gets resolved

**Comment by catherinebirney** [comment]
@dpgraham4401 Can you try again? The update to the stewi version should fix the error. 

I created a [new wiki page ](https://github.com/USEPA/flowsa/wiki/Install-for-Developers)for installing flowsa as a developer. Following the install instructions as they were written might have also contributed to the errors.

**Comment by dpgraham4401** [comment]
@catherinebirney  the [current install instructions](https://github.com/USEPA/flowsa/wiki/Install-&-Run) still point to the v0.2.1 tag which includes a requirement for stEWI 0.9.8 :)

```
ERROR: Cannot install flowsa and flowsa==0.2.1 because these package versions have conflicting dependencies.
The conflict is caused by:
    flowsa 0.2.1 depends on esupy 0.1.7 (from git+https://github.com/USEPA/esupy@v0.1.7#egg=esupy)
    stewi 0.9.8 depends on esupy 0.1.2 (from git+git://github.com/USEPA/esupy@v0.1.2#egg=esupy)

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

I can successfully install by removing the tag from like such 
```pip install git+https://github.com/USEPA/flowsa```

but when I try to import I'm getting additional error(s), I will try to look into this tonight but have got to run now.

**Comment by catherinebirney** [comment]
@dpgraham4401 I released v0.3 of flowsa (and updated the wiki install instructions). Let me know if continuing to get errors.
